cpp/tools/clustertool.h

```diff
@@ -10,9 +18,15 @@ class ClusterTool {
   // Initialise a fresh log cluster:
   //  - Creates /serving_sth containing a new STH of size zero
   //  - Creates the /cluster_config entry.
-  static util::Status InitLog(TreeSigner<Logged>* tree_signer,
+  static util::Status InitLog(const ct::ClusterConfig& cluster_config,
+                              TreeSigner<Logged>* tree_signer,
                               ConsistentStore<Logged>* consistent_store);
 
+  // Sets the cluster config
+  static util::Status SetClusterConfig(
+      const ct::ClusterConfig& cluster_config,
+      ConsistentStore<Logged>* consistent_store);
```

Comment:
I was already feeling that in the previous pull request: either those should be free functions, or that class should start pulling its weight a bit and have something (like the tree signer and the consistent store)? Even then, that wouldn't really be state, just all const stuff, so I'd probably favour free functions? But I'll leave it to you...